firmware/application/apps/ui_adsb_rx.hpp

```diff
@@ -343,8 +343,8 @@ class ADSBRxView : public View {
    private:
     RxRadioState radio_state_{
         2500000 /* bandwidth */,
-        2000000 /* sampling rate */
-    };
+        2000000 /* sampling rate */,
+        ReceiverModel::Mode::SpectrumAnalysis};
```

Comment:
In discussion with @Brumi-2021 we're wondering why ADSB is using "SpectrumAnalysis" mode which has no tuning offset and therefore incurs a signal "spike".  Leaving it at the default "NarrowbandFMAudio" seems to work fine in my testing, so perhaps this line doesn't need to be changed.

Reply:
Well , good point,  we both have that doubt, but as  @kallanreed answered me , in this PR, is just moving the same SPEC settings from implicit  file , to the Radiostate.  
Then let's agree by now , and continue that separated discussion & investigation outside of this PR.

Reply:
@Brumi-2021 - thoughts on this change? Could either leave this as NFM or you had suggested Capture. I'm tempted to use the default that Mark tested.

Reply:
Looks like the code has been using SPEC for at least 2 years, so I'd go ahead and leave it as SPEC for this PR.  We'll take this one offline; sorry to get side-tracked.